packages/grafana-ui/src/components/Logs/LogRowContext.tsx

```diff
@@ -197,6 +197,19 @@ export const LogRowContext: React.FunctionComponent<LogRowContextProps> = ({
   onLoadMoreContext,
   hasMoreContextRows,
 }) => {
+  const handleEscKeyDown = (e: KeyboardEvent): void => {
+    if (e.keyCode === 27) {
+      onOutsideClick();
+    }
+  };
+
+  useEffect(() => {
```

Comment:
Can we add unit test for this? Tbh not sure how easy it is to simulate native events so if it would be problematic let me know.

Reply:
@aocenas Played with that on Friday but I found it kinda tricky, not sure how to check if a specific event listener exists on the element without setting up additional attributes (or state variables). Any tips?

Reply:
Yeah no idea right now, so lets not block this on that we can figure it out later.